pages/services/dse/3.0.0-6.7.2/release-notes/index.md

```diff
@@ -0,0 +1,219 @@
+	- Now users can issue nodetool commands against all nodes in their cluster via command:
+	```
+	dcos datastax-dse plan start nodetool-ser \
+	  -p NODETOOL_CONNECTION_OPTS='<connection_options>'  \
+	  -p NODETOOL_SUBCOMMAND='<subcommand>'  \
+	  -p NODETOOL_CMD_ARGS='<subcommand_arguments>'
+	```
+	- For example:
+	```
+	dcos datastax-dse plan start nodetool-ser \
+	  -p NODETOOL_CONNECTION_OPTS='-p 7199'  \
+	  -p NODETOOL_SUBCOMMAND='upgradesstables'  \
+	  -p NODETOOL_CMD_ARGS='-a'
+	```
+- Upgrading the SDK from the 0.4x series to 0.5x comes with significant framework and scheduler improvements such as:
+	- Adding [multi-service support](https://github.com/mesosphere/dcos-commons/releases/tag/0.50.0) and region-/zone-awareness
+	- [Scheduler metrics](https://github.com/mesosphere/dcos-commons/releases/tag/0.52.0)
+	- Improving logging, debuggability, and bug fixes
+	- Please read SDK-related release notes to learn more: https://github.com/mesosphere/dcos-commons/releases
+
+## Upgrading your cluster from DSE 5.1 to 6.7
+Due to the complexity of upgrading to DSE 6.7, it is highly advised that you attempt the upgrade on a test cluster before upgrading in your production environment.
+
+In order to upgrade your cluster from DSE 5.1 to DSE 6.7 you must:
+- Review official DSE upgrade docs: https://docs.datastax.com/en/upgrade/doc/upgrade/datastax_enterprise/upgdDSE51to67.html
+  - follow instructions in the [Upgrade restrictions and limitations](https://docs.datastax.com/en/upgrade/doc/upgrade/datastax_enterprise/upgdDSE51to67.html#Upgraderestrictionsandlimitations)
+  - if using CFS, copy your data from it
+  - Using `cqlsh`, drop CFS keyspaces: `cfs` and `cfs_archive`
+- Run the following command to upgrade your DSE package: 
+  ```
+  dcos datastax-dse update start --package-version=3.0.0-6.7.2
+  ```
+	- This will run a script to drop `COMPACT_STORAGE` from all keyspaces and then upgrade each DSE node to version `6.7.2`
+	<p class="message--important"><strong>IMPORTANT: </strong>Before upgrading, make sure the agent running <code>dse-0-node</code> has more than 0.25 CPU available. This is required to run the <code>compact-storage</code> task. You may have to move other services running on that agent to accommodate.</p>
+
+- After upgrade succeeds, run the following command to convert sstables to the proper version:
+	```
+	dcos datastax-dse plan start nodetool-ser \
+	  [-p NODETOOL_CONNECTION_OPTS='-p 7199']  \  ## optional
+	  -p NODETOOL_SUBCOMMAND='upgradesstables'  \
+	  -p NODETOOL_CMD_ARGS='-a'
+	```
+	<p class="message--important"><strong>IMPORTANT: </strong>This action cannot be undone and you should plan for increased load activity on your cluster. This task should be scheduled for off-peak hours. Should any problems arise, `pause` the plan and investigate.
+- After DSE upgrade, you can upgrade OpsCenter with the following command: 
```

Comment:
We need to move OpsCenter upgrade before previous item (`upgradesstables`) - upgrading of SSTables could take a time, and it's "optional" - DSE will work with "old" format, maybe not 100% effective although